### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Aziz-Prithibee13** [vercel/community#3252] [Help]
I am trying to add an graphql on my node express js server side. I create it and it do much better in my local machine. But when I deploy it in vercel it showing this error.

{
  "error": {
    "code": "504",
    "message": "An error occurred with your deployment"
  }
}

I will use this server on my react-native project. In web I handle it cause it was same machine. But in mobile app, I cant do this any more. 

Here my repository. https://github.com/Aziz-Prithibee13/movie-mania-server

Help to resolve this mess.

#TIA.

**Comment by amyegan** [comment]
Hey @Aziz-Prithibee13. A 504 means the serverless function took to long to respond. You can find some troubleshooting tips about this error at https://vercel.com/docs/errors/application-error-codes#504:-function_invocation_timeout

**Reply by Aziz-Prithibee13** [reply]
![Screenshot_20230715-025614.png](https://github.com/vercel/community/assets/137806759/36dae31c-371a-44d5-9049-f6a0cba20b8c)

Showing this in logs route.

**Reply by Aziz-Prithibee13** [reply]
![Screenshot_20230715-145053.png](https://github.com/vercel/community/assets/137806759/575dbb36-e05c-46f2-bcd5-8e8fc84ed640)

Got the same 504 error after using an edge function.
